### ISSUE TITLE
variableOffsetPP

### ISSUE THREAD

**Issue by ggalibert** [aodn/imos-toolbox#432]
variableOffsetPP needs to be improved so that a pop-up window enables the user to document scale and offset for any instrument loaded, possibly on multiple parameters. This also needs to be captured for re-processing.

**Comment by sspagnol** [comment]
Can I ask for a change in usage : I recently used variableOffsetPP to fix an offset in PRES_REL variable for an RDI ADCP. This worked well. But didn't realize that this then put an entry into variableOffsetPP.txt so the next lot of data I loaded up had an offset applied to *ALL* PRES_REL variables. So I would like to not have variableOffsetPP.txt used at all and all usage of variableOffsetPP be captured per instrument file for reprocessing.
